lib/landing_pages/import_export/zip_exporter.rb

```diff
@@ -14,33 +14,14 @@ def export_to_folder
 
     about = {}
 
-    %w(name path remote).each do |attr|
+    LandingPages::Page.pages_attrs.each do |attr|
       value = @page.send(attr)
-
-      next unless value.present?
-
-      if attr === "theme_id"
```

Comment:
@cabidop Why is this logic being removed?

Reply:
It was not being used: if you check line 17 in the same file the only considered attributes are `name`, `path` and `remote`, so the `theme_id` and the `group_ids` conditions were never checked.

This also meant that these two attributes (plus `parent_id` and `category_id`) where never added to the exported data, so using `LandingPages::Page.pages_attrs` instead of `%w(name path remote)` for the loop makes that possible now.

Reply:
Ok, but if we're now also checking `theme_id` and `group_ids` won't we need the logic concerning their appropriate `value`?

Reply:
With the update on the importer the "raw" values for the attributes (int for the `theme_id` and array of ints for the `group_ids`) can be handled directly, without further conversions between types.

Reply:
Ok